boards/arm/96b_wistrio/96b_wistrio.dts

```diff
@@ -90,7 +90,7 @@
 	status = "okay";
 	cs-gpios = <&gpiob 0 GPIO_ACTIVE_LOW>;
 
-	lora: sx1276@0 {
+	lora: lora@0 {
```

Comment:
Is there a need to keep the first lora occurence now ?

Reply:
Sorry, I don't get what you are saying.

Reply:
The first is the nodelabel, the second is the node name. They are both needed.